frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -109,13 +115,18 @@ const ActivitySummary = ({
       <Helmet>
         <title>Activity summary</title>
       </Helmet>
+      <legend className="margin-top-1">
+        <span className="smart-hub--form-required font-family-sans font-ui-xs">* </span>
+        indicates required field
+      </legend>
       <Fieldset className="smart-hub--report-legend margin-top-4" legend="Who was the activity for?">
         <div id="activity-for" />
         <div className="margin-top-2">
           <FormItem
             label="Was this activity for a recipient or other entity?"
             name="activityRecipientType"
             fieldSetWrapper
+            requiredText="*"
           >
             <Radio
               id="category-recipient"
```

Comment:
can we mark the first radio button ```required```? I think just putting the required prop on here will do it.

Reply:
Will add required.